cpp/src/command_classes/CommandClasses.cpp

```diff
@@ -202,6 +203,7 @@ void CommandClasses::RegisterCommandClasses
 	cc.Register( ApplicationStatus::StaticGetCommandClassId(), ApplicationStatus::StaticGetCommandClassName(), ApplicationStatus::Create );
 	cc.Register( Association::StaticGetCommandClassId(), Association::StaticGetCommandClassName(), Association::Create );
 	cc.Register( AssociationCommandConfiguration::StaticGetCommandClassId(), AssociationCommandConfiguration::StaticGetCommandClassName(), AssociationCommandConfiguration::Create );
+	cc.Register( AVSimple::StaticGetCommandClassId(), AssociationCommandConfiguration::StaticGetCommandClassName(), AVSimple::Create );
```

Comment:
Just a Nitpick - Can we rename to SimpleAV to match the specs rather than AVSimple?

Reply:
(And rename the Files etc/Class names etc etc etc)